### PR TITLE
CelebA Handling

### PR DESCRIPTION
**Pull request by usert5432**
Hi @YHRen,

I am opening a PR that changes CelebA handling in `uvcgan`. I have dropped all the CelebA download code. And, added a bunch of notices about the need to recreate the CelebA datasets. Could I ask you please to take a look over this PR?

Also, I made a change to `examples/predict.py` (d9cc35ebbdc478f6024f13b4b09312f91132d289), since I do not think it can work as is.

Related: https://github.com/LS4GAN/uvcgan/issues/23, https://github.com/LS4GAN/uvcgan/issues/21 
Closes: https://github.com/LS4GAN/uvcgan/issues/22

### CODE REVIEW

File: scripts/download_dataset.sh [hunk 1]
```diff
@@ -144,56 +166,33 @@ move_files ()
...
     exec_or_die mkdir -p "${path}/train" "${path}/val"
 
-    exec_or_die mv "${unzipped_path}/trainA" "${unzipped_path}/trainB" \
+    echo "Copying files..."
+
+    exec_or_die cp -r "${path_m2f}/trainA" "${path_m2f}/trainB" \
         "${path}/train/"
 
-    exec_or_die mv "${unzipped_path}/testA" "${unzipped_path}/testB" \
+    exec_or_die cp -e "${path_m2f}/testA" "${path_m2f}/testB" \
```
Comment (YHRen) [inline scripts/download_dataset.sh:194]: `-e` -> `-r` ?

**Review by YHRen — COMMENTED** [review]
@usert5432  Thank you so much! Everything looks good except a potential typo. Once you review and/or fix it, feel free to merge.

File: scripts/download_dataset.sh (hunk 1 above)
Comment (usert5432) [reply scripts/download_dataset.sh]: Ah, Thank you, @YHRen!

**Comment by usert5432** [comment]
Merged. Thank you @YHRen for the review!

**Pull request merged**